Templates/Integrations/DataEnrichmentThreatIntelligence/DataEnrichmentThreatIntelligence.py

```diff
@@ -0,0 +1,376 @@
+        }
+        context.update(dbot_entry)
+        human_readable = tableToMarkdown(title, context_entry, removeNull=True)
+        return human_readable, context, raw_response
+    else:
+        return f'{INTEGRATION_NAME} - No results found for URL: {url}', {}, raw_response
+
+
+@logger
+def search_file_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    """Searching for given file hash.
+
+    Args:
+        client: Client object
+        args: Usually demisto.args()
+
+    Returns:
+        Outputs
+    """
+    file_hash = args.get('file', '')
+    try:
+        threshold = int(args.get('threshold'))
+    except TypeError:
+        threshold = None
+    raw_response = client.search_file(file_hash)
+    results = raw_response.get('result')
+    if results:
+        result = results[0]
+        title = f'{INTEGRATION_NAME} - Analysis results for file hash: {file_hash}'
+        context_entry = {
+            'ID': result.get('id'),
+            'Severity': result.get('severity'),
+            'MD5': result.get('md5'),
+            'SHA1': result.get('sha1'),
+            'SHA256': result.get('sha256'),
+            'SSDeep': result.get('ssdeep'),
+            'Description': result.get('description')
+        }
+        # Gets DBot score
+        score = client.calculate_dbot_score(result.get('severity'), threshold=threshold)
+        # Building a score for DBot
+        dbot_score = [
+            {
+                'Indicator': result.get(hash_name),
+                'Type': 'hash',
+                'Vendor': f'{INTEGRATION_NAME}',
+                'Score': score
+            } for hash_name in FILE_HASHES if result.get(hash_name)
+        ]
+        context = {
+            outputPaths['dbotscore']: dbot_score,
+            f'{INTEGRATION_CONTEXT_NAME}(val.ID && val.ID === obj.ID)': context_entry
+        }
+
+        if score == 3:  # If file is malicious, adds a malicious entry
+            context[outputPaths['file']] = [{
+                hash_name.upper(): raw_response.get(hash_name),
+                'Malicious': {
+                    'Vendor': f'{INTEGRATION_NAME}',
+                    'Description': raw_response.get('description')
+                }
+            } for hash_name in FILE_HASHES if raw_response.get(hash_name)]
+        human_readable = tableToMarkdown(title, context_entry, removeNull=True)
+        return human_readable, context, raw_response
+    else:
+        return f'{INTEGRATION_NAME} - No results found for file hash: [{file_hash}', {}, raw_response
+
+
+@logger
+def search_domain_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    """Gets a job from the API. Used mostly for polling playbook.
+
+    Args:
+        client: Client object
+        args: Usually demisto.args()
+
+    Returns:
+        Outputs
+    """
+    url = args.get('domain')
+    raw_response = client.get_domain(url)
+    results = raw_response.get('result')
+    if results:
+        result = results[0]
```

Comment:
why only the first?

Reply:
because the api will return one result per call with 1 domain.